src/project/git/GitCommandGitProject.ts

```diff
@@ -102,6 +102,7 @@ export class GitCommandGitProject extends NodeFsLocalProject implements GitProje
                         private credentials: ProjectOperationCredentials, release: ReleaseFunction,
                         public provenance?: string) {
         super(id, baseDir, release);
+        this.branch = id.sha;
```

Comment:
So the sha is a branch?

Reply:
yeaaaahhh. usually.

what is the name for "sha or branch" ? the git docs seem to use the word "commit" for that, but it's more of a "commit pointer" or "commit identifier" kind of thing

Reply:
ref?

Reply:
a commit SHA is not technically a ref, i don't think